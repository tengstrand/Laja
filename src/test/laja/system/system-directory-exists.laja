
#write "target/test-output/system-directory-exists.out"
  #if (directoryExists("{.}"))
    This directory exists!
  #end
  #if (directoryExists("{.}/x"))
    The directory x exists
  #else
    The directory x does not exist
  #end
  #if (directoryExists("{.}/system-directory-exists.laja"))
    This is a file, not a directory
  #else
    The directory does not exist (it's a file)
  #end
#end
