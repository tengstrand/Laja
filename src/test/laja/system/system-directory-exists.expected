    This directory exists!
    The directory x does not exist
    The directory does not exist (it's a file)
